rust/core-lib/src/styles.rs

```diff
@@ -184,6 +185,7 @@ impl ThemeStyleMap {
             theme,
             default_style,
             map: HashMap::new(),
+            path_map: BTreeMap::new(),
```

Comment:
`path_map` vs `state` is definitely something I haven't considered well, and is mostly part of the old theme load implementation. I think I'll change it so `path_map` can be used as the `state` as well.

The main reason I picked a `BTreeMap` was because I wanted the themes list to be sorted alphabetically by key (i.e the theme names).